docs/src/criticalup_toml.rst

```diff
@@ -87,6 +87,18 @@ channels page <https://releases.ferrocene.dev/ferrocene/index.html>`_.
     release = "stable-24.05.0"
     # ...
 
+**Getting the latest release in a channel**
+
+Sometimes you may want the latest release for a given channel. You can use
```

Comment:
This section doesn't match the tone or perspective of other entries, can we align it?

Reply:
I tried to match tone.